diff --git a/app/Http/Controllers/PostController.ts b/app/Http/Controllers/PostController.ts
index f837f50..bed64cd 100644
--- a/app/Http/Controllers/PostController.ts
+++ b/app/Http/Controllers/PostController.ts
@@ -93,7 +93,10 @@ export default class PostController extends Controller {
     const data = await this.validateBody(PostPayloadSchema)
     await Post.update({ id: post.id }, data)
 
-    return this.redirect(`/posts/${post.id}`)
+    return new Response(null, {
+      status: 302,
+      headers: { Location: `/posts/${post.id}` },
+    })
   }
 
   async destroy(): Promise<Response> {
