diff --git b/app/Http/Resources/PostResource.ts a/app/Http/Resources/PostResource.ts
index 60945ca..f798f49 100644
--- b/app/Http/Resources/PostResource.ts
+++ a/app/Http/Resources/PostResource.ts
@@ -17,12 +17,13 @@ export interface PostResourceData extends Record<string, unknown> {
 }
 
 export class PostResource extends Resource<PostRecord | PostWithAuthor> {
-  toArray() {
+  toArray(): PostResourceData {
     const post = this.resource
 
     return {
       id: post.id as number,
       title: post.title as string,
+      excerpt: post.excerpt as string,
       body: post.body as string,
       authorId: post.authorId as number,
       createdAt: new Date(post.createdAt as string | Date).toISOString(),
