diff --git a/app/Http/Resources/PostResource.ts b/app/Http/Resources/PostResource.ts
index f798f49..60945ca 100644
--- a/app/Http/Resources/PostResource.ts
+++ b/app/Http/Resources/PostResource.ts
@@ -17,13 +17,12 @@ export interface PostResourceData extends Record<string, unknown> {
 }
 
 export class PostResource extends Resource<PostRecord | PostWithAuthor> {
-  toArray(): PostResourceData {
+  toArray() {
     const post = this.resource
 
     return {
       id: post.id as number,
       title: post.title as string,
-      excerpt: post.excerpt as string,
       body: post.body as string,
       authorId: post.authorId as number,
       createdAt: new Date(post.createdAt as string | Date).toISOString(),
