diff --git a/app/Http/Controllers/PostController.ts b/app/Http/Controllers/PostController.ts
index cccd770..ddfa7bb 100644
--- a/app/Http/Controllers/PostController.ts
+++ b/app/Http/Controllers/PostController.ts
@@ -35,12 +35,12 @@ export default class PostController extends Controller {
 
     const posts = await Post.newQuery()
       .with('author')
+      .where('published', true)
       .where((q) => {
         for (const keyword of keywords) {
           const pattern = `%${keyword}%`
           q.orWhere('title', 'like', pattern).orWhere('excerpt', 'like', pattern)
         }
-        q.orWhere('published', true)
       })
       .orderBy('id', 'desc')
       .limit(limit)
