diff --git a/app/Http/Controllers/PostController.ts b/app/Http/Controllers/PostController.ts
index f837f50..5296ec4 100644
--- a/app/Http/Controllers/PostController.ts
+++ b/app/Http/Controllers/PostController.ts
@@ -90,8 +90,7 @@ export default class PostController extends Controller {
     const post = await Post.findOrFail(id)
     await this.authorize('update', [Post, post])
 
-    const data = await this.validateBody(PostPayloadSchema)
-    await Post.update({ id: post.id }, data)
+    await Post.forceUpdate({ id: post.id }, await this.request.json())
 
     return this.redirect(`/posts/${post.id}`)
   }
